template-parts/content/pagination.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Template part for displaying a pagination
+ *
+ * @package wp_rig
+ */
+
+namespace WP_Rig\WP_Rig;
+
+if ( is_search() ) {
+	the_posts_pagination(
+		array(
+			'mid_size'           => 2,
+			'prev_text'          => _x( 'Previous', 'previous set of search results', 'wp-rig' ),
+			'next_text'          => _x( 'Next', 'next set of search results', 'wp-rig' ),
+			'screen_reader_text' => __( 'Search results navigation', 'wp-rig' ),
+		)
+	);
+} else {
```

Comment:
Let's remove the if else here, and use `the_posts_pagination()` everywhere.

Reply:
@felixarntz Shouldn't we add a template part for post navigation named `navigation.php` where we would have `the_post_navigation()` (used in `entry.php` for singular posts) and `the_posts_navigation()`;?

Reply:
I don't think we need it. The point of this template is to provide a single point to change that. If we had two separate templates, you would again need to change the reference to the template everywhere. The choice of `pagination` for the name is solely because we think it is more appropriate than `navigation` (because that is more generic and can mean anything), it doesn't actually mean this file _has_ to call `the_posts_pagination()`. It only means it is responsible for rendering the pagination, in whichever way the theme author prefers.

Reply:
Thanks you for the explanation. The code is ready to merge.